src/pages/Account/Venue/VenueMapEdition/PartyMapContainer.tsx

```diff
@@ -1,24 +1,24 @@
 import React, { useMemo } from "react";
+import { Room } from "types/rooms";
 
 import { ExtractProps } from "types/utility";
-import { PartyMapVenue } from "types/venues";
 
 import { Container, SubVenueIconMap } from "./Container";
 
 type PartyMapContainerProps = Omit<
   ExtractProps<typeof Container>,
   "otherIcons"
 > & {
-  venue: PartyMapVenue;
+  rooms: Room[];
   currentRoomIndex?: number;
 };
 
 export const PartyMapContainer: React.FC<PartyMapContainerProps> = (props) => {
-  const { currentRoomIndex, venue, ...rest } = props;
+  const { currentRoomIndex, rooms, ...rest } = props;
```

Comment:
Is `props` ever passed on to anything else (doesn't look like it)? If not, can we use our standard pattern here?

```typescript
export const PartyMapContainer: React.FC<PartyMapContainerProps> = ({
  currentRoomIndex,
  rooms,
  ...rest
}) => {
```